### PR TITLE
Update to .NET Core 1.0.10, 1.1.8, and 2.0.6

### PR DESCRIPTION
**Pull request by natemcmaster**
Changes:
 - Update bundled .NET Core runtime to 1.0.10, 1.1.8, and 2.0.6
 - Update SDK to 2.1.101
 - Add new tags for these new runtimes and update `<major>.<minor>` tags for the patch updates

Heads up: CI is expected to fail until https://github.com/dotnet/dotnet-docker/pull/422 is completed.

### CODE REVIEW

File: README.aspnetcore.md [hunk 1]
```diff
@@ -10,23 +10,23 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build`](https:
 
 # Supported Linux amd64 tags
 
-- [`1.0.9-jessie`, `1.0.9`, `1.0` (*1.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/runtime/Dockerfile)
+- [`1.0.10-jessie`, `1.0.10`, `1.0` (*1.0/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.0/jessie/runtime/Dockerfile)
 - [`1.1.6-jessie`, `1.1.6`, `1.1`, `1` (*1.1/jessie/runtime/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/jessie/runtime/Dockerfile)
```
Comment (MichaelSimons) [inline README.aspnetcore.md]: I was expecting to see a version rev for the two 1.1.* images.

Comment (natemcmaster) [reply README.aspnetcore.md]: Good catch, yes this should bump.

**Comment by natemcmaster** [comment]
@dotnet-bot test ci please

**Comment by MichaelSimons** [comment]
FYI not all images are available yet.

**Comment by MichaelSimons** [comment]
@dotnet-bot test ci please

**Comment by MichaelSimons** [comment]
Hmm, maybe I don't have the magic powers to kick CI.  Regardless the microsoft/dotnet images are available now.

**Comment by natemcmaster** [comment]
@dotnet-bot test Windows SAC2016 2.0/nanoserver-sac2016 Build
@dotnet-bot test Windows SAC2016 1 Build
@dotnet-bot test Windows 1709 2.0/nanoserver-1709 Build
@dotnet-bot test Linux 2.0 Build 
@dotnet-bot test Linux 1 Build

**Comment by natemcmaster** [comment]
For some reason, the  "dotnet-bot test ci please" comment isn't triggering all builds. @mmitche any ideas?

**Comment by MichaelSimons** [comment]
I just realized it was the wrong cmd.  You want to say `dotnet-bot test please`.  `dotnet-bot test ci please` is what triggers the Gen CI job.

**Comment by natemcmaster** [comment]
Ah, that makes more sense. Didn't realize there was a difference.

**Comment by natemcmaster** [comment]
@dotnet-bot test Windows 1709 2.0/nanoserver-1709 Build
@dotnet-bot test Windows SAC2016 2.0/nanoserver-sac2016 Build

**Comment by natemcmaster** [comment]
@dotnet-bot test Windows SAC2016 1 Build

**Pull request merged**